google-cloud-spanner/src/main/java/com/google/cloud/spanner/AsyncResultSetImpl.java

```diff
@@ -449,6 +458,27 @@ private void startCallbackWithBufferLatchIfNecessary(int bufferLatch) {
     }
   }
 
+  private class InitiateStreamingRunnable implements Runnable {
+
+    @Override
+    public void run() {
+      try {
+        // This method returns true if the underlying result set is a streaming result set (e.g. a
+        // GrpcResultSet).
+        // Those result sets will trigger initiateProduceRows() when the first results are received.
+        // Non-streaming result sets do not trigger this callback, and for those result sets, we
+        // need to eagerly
+        // start the ProduceRowsRunnable.
```

Comment:
nit: move this line up to the previous line (this was probably automatically mangled by the code formatter)